README.md

```diff
@@ -1,5 +1,7 @@
 # Language Independent Interface Types For OpenTelemetry
 
+![Build Check](https://github.com/open-telemetry/opentelemetry-proto/workflows/Build%20Check/badge.svg)
```

Comment:
Traditionally these badges are linked to details about the build, such as the /Actions tab in the repo (with a filter on the branch name).